app/representers/space_representer.rb

```diff
@@ -27,5 +27,11 @@ class SpaceRepresenter < Roar::Decorator
     property :admin_id
     property :name
     property :picture
+
+    property :plan_details,
```

Comment:
I'll play with it a bit.  One challenge actually was that the user_options would be messy if it depended on the user.  It smelled bad to me, can consider it again though.